### PR TITLE
[VEN-2560][VEN-2563]: Enable Prime on Arbitrum

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2560

### CODE REVIEW

File: multisig/proposals/arbitrumsepolia/vip-007/index.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: COMPTROLLER_CORE,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
```
Comment (chechu) [inline multisig/proposals/arbitrumsepolia/vip-007/index.ts:121]: I think we don't need this command yet. We'll execute in the VIP setting the Prime markets. I suppose markets from the Core pool comptroller will be added, but potentially only markets from other pools could be added, and in that case we wouldn't need the Prime contract set on the Core pool comptroller

Comment (Debugger022) [reply multisig/proposals/arbitrumsepolia/vip-007/index.ts]: Got it. https://github.com/VenusProtocol/vips/pull/264/commits/ca43ab0b478ce3d97555ab0b0af5f0b2e36e3122

**Review by chechu — COMMENTED** [review]
TX for Arbitrum Sepolia lgtm

File: multisig/proposals/arbitrumone/vip-007/index.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      params: [
+        0, // irrevocable
+        0, // revocable
+      ],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "pauseFundsTransfer()",
+      params: [],
+    },
```
Comment (chechu) [inline multisig/proposals/arbitrumone/vip-007/index.ts:132]: We should resume the XVSVault at the end, as it was made at [007 on Ethereum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/ethereum/vip-007/index.ts#L117-L121)

`
    {
      target: XVS_VAULT_PROXY,
      signature: "resume()",
      params: [],
    },
`

**Review by GitGuru7 — APPROVED** [review]

File: multisig/proposals/arbitrumone/vip-007/index.ts (hunk 2 above)
Comment (Debugger022) [reply multisig/proposals/arbitrumone/vip-007/index.ts:132]: Done https://github.com/VenusProtocol/vips/pull/264/commits/56a49e0cf7511dda2a4edc9eae6eb88b17d76a19

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/320?chainId=56

**Pull request merged**